app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,18 @@
   </head>
 
   <body>
+  <div style='text-align:center'>
+<% flash.each do |type, message| %>
+    <% if type == "success" %>
+    <%= content_tag :div, sanitize(message), class:"alert-success" %>
```

Comment:
Great - I think Thursday we can go back and refactor, and review test coverage that we've covered so far. In this early stage of the project, I'd like to push forward with functionality and refactor to get to learn the app we've built even better before moving on to the next phase. Does that sound good @AELSchauer @JonathanSR @alfosco ?